src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,814 @@
+              toolbar--template--HasSortButton="true"
+              toolbar--template--HasOverflowMenu="true"
+              toolbar--template--HasOverflowMenuSecondButton="true"
+              toolbar--template--OverflowButton1Text="Generate"
+              toolbar--template--OverflowButton2Text="Deploy"
+              toolbar--template--filterText="Name"}}
+            {{> divider}}
+            {{> tabs--table}}
+            {{#> pagination pagination--modifier="pf-m-bottom"}}
+              {{> pagination-options-menu id=(concat tabs--page-wrapper--id '-footer-pagination') options-menu--IsText="true" pagination-options-menu--modifier="pf-m-top"}}
+              {{> pagination-nav-content}}
+            {{/pagination}}
+          {{/drawer-content}}
+
+          <!-- Panel -->
+          {{#> drawer-panel drawer-panel--modifier="pf-m-width-33 pf-m-width-33-on-xl"}}
+            {{> tabs--panel-header tabs--panel-header--title="Node 2" tabs--panel-header--sub-title='<a href="#">siemur/test-space</a>'}}
+            {{#> drawer-body drawer-body--modifier="pf-m-no-padding"}}
+              {{> primary-detail-panel-tabs primary-detail-panel-tabs--Has3="true" primary-detail-panel-tabs--modifier="pf-m-no-border-bottom pf-m-inset-md pf-m-inset-sm-on-md"}}
+            {{/drawer-body}}
+            {{#> drawer-body primary-detail-template--id=(concat tabs--page-wrapper--id '-tabs')}}
+              {{> primary-detail-panel-tab-content progress--modifier="pf-m-sm" primary-detail-panel-tab-content--HasLabels="true" primary-detail-panel-tab-content--Has3="true"}}
+            {{/drawer-body}}
+          {{/drawer-panel}}
+        {{/drawer-main}}
+      {{/drawer}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-node-connectors-link" id="' tab-content--id '-node-connectors-panel"')}}
+      {{#> content}}
+        <p>Node connectors content</p>
+      {{/content}}
+    {{/tab-content}}
+  {{/page-main-section}}
+{{/tabs--page-wrapper}}
+```
+
+### Modal tabs
+```hbs isFullscreen
+{{#> example-wrapper example-wrapper--id="modal-tabs-example"}}
+  {{#> modal}}
+    {{#> backdrop}}
+      {{#> bullseye}}
+        {{#> modal-box modal-box--modifier="pf-m-sm" modal-box--attribute=(concat 'aria-labelledby="' example-wrapper--id '-modal-title" aria-describedby="' example-wrapper--id '-modal-description"')}}
+          {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close dialog"'}}
+            <i class="fas fa-times" aria-hidden="true"></i>
+          {{/button}}
+          {{#> modal-box-header}}
+            {{#> modal-box-title modal-box-title--attribute=(concat 'id="' example-wrapper--id '-modal-title"')}}
+              Modal title
+            {{/modal-box-title}}
+          {{/modal-box-header}}
+          {{#> modal-box-body modal-box-body--attribute=(concat 'id="' example-wrapper--id '-modal-description"')}}
+            {{#> grid grid--modifier="pf-m-gutter"}}
```

Comment:
Why are we using grid here but flex in https://patternfly-pr-3914.surge.sh/components/tabs/html-demos/nested-tabs/?